scripts/gen-cli-data.py

```diff
@@ -36,24 +53,26 @@ def generate(tool, storage='', sampling=''):
     print(cmd)
     if os.system(cmd) != 0:
         sys.exit(1)
-    if storageOrSampling:
+
+    if feature_name:
         os.rename(
-            '{}/data/cli/{}/{}.yaml'.format(output_path, jaeger_ver, tool),
-            '{}/data/cli/{}/{}-{}.yaml'.format(output_path, jaeger_ver, tool, storageOrSampling)
-        )
+            f'{output_path}/data/cli/{jaeger_ver}/{tool}.yaml',
+            f'{output_path}/data/cli/{jaeger_ver}/{tool}-{feature_name}.yaml')
 
 
 for tool in cfg['tools']:
     tool_cfg = cfg[tool]
-    storage_types=tool_cfg['storage']
-    sampling_types=tool_cfg['sampling']
-    if storage_types:
```

Comment:
These `if`s were redundant as a "zero" value of a list just means the body of the `for` loop won't be executed.